CSS/style.css

```diff
@@ -549,6 +708,83 @@ h3 {
   padding-left: 195px;
 }
 
+@media only screen and (min-width: 995px) {
+  .main-about {
+    margin-top: -1220px;
+    width: 1440px;
+    height: 1120px;
+    background-image: url("../images/illustration-about\ me.png");
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: 100px;
+  }
+
+  .abt-frame {
+    display: none;
+  }
+
+  .about-me {
+    margin-left: 142px;
+  }
+
+  .abt-text {
+    width: 566px;
+    margin-top: 24px;
+    margin-bottom: -80px;
+  }
+
+  #abt-line {
+    margin-top: 130px;
+    margin-right: 142px;
+    width: 1156px;
+  }
+
+  .category li {
+    display: inline;
+    align-items: center;
+  }
+
+  .category .fram {
+    margin-left: 500px;
+    margin-top: -322px;
+  }
+
+  #fram-icon {
+    margin-top: -400px;
+    margin-bottom: 287px;
+    margin-left: 330px;
+  }
+
+  .category .skill {
+    margin-top: -390px;
+    margin-left: 1000px;
+  }
+
+  #skill-icon {
+    margin-top: -288px;
+    margin-bottom: 355px;
+    margin-left: 750px;
+  }
+
+  .category {
+    margin-top: 60px;
+    margin-left: 100px;
+    margin-right: 153px;
+  }
+
+  .category li .framework {
+    margin-left: 450px;
+  }
+
+  .category li .skills {
+    margin-left: 930px;
+  }
+
+  .category li ul {
+    align-items: flex-start;
+  }
+}
+
 input[type="email"] {
   display: block;
```

Comment:
- [ ] You can make the form more distinctive by adding a frame(border) to it, and also by targeting the `class` instead of the `tag`

Reply:
I made it with the attribute submit, mail, name, and comment ( ID attributes), and like that my form is distinctive. But, thanks for your advice.